drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1616,6 +1653,32 @@ function DragAndDropBlock(runtime, element, configuration) {
                 $container.addClass('dragging');
                 $root.find('.target .zone').first().focus();
             }
+
+            // Enable sliding in accessibility mode
+            if (DragAndDropBlock.isItemSizingFixed()) {
+                var $item = $(this);
+                var itemNumber = $item[0].attributes['data-value'].value;
+                var maxItems = $container.find('.option[draggable=true]').length;
+
+                if (isBrowsingForwardsInAccessibility(evt)) {
+                    if (itemNumber>1 && itemNumber%8 === 7) {
+                        evt.preventDefault();
```

Comment:
As there is no else condition, every block has a condition on it. If the condition is not met, the default functionality should be allowed to work.
Hence, putting the line in outer if would break functionality and wont let the user tab through items in item bank.

Reply:
You're right. I think I misread the next block of code.